web/ui/react-app/src/DataTable.tsx

```diff
@@ -59,45 +64,69 @@ class DataTable extends PureComponent<QueryResult> {
 
     let rows: ReactNode[] = [];
     let limited = false;
-    switch(data.resultType) {
+    switch (data.resultType) {
       case 'vector':
-        rows = (this.limitSeries(data.result) as InstantSample[])
-          .map((s: InstantSample, index: number): ReactNode => {
-            return <tr key={index}><td><SeriesName labels={s.metric} format={false}/></td><td>{s.value[1]}</td></tr>;
```

Comment:
Since this line length was below 125 chars, I guess there's also an explicit rule that disallows putting any nested JSX tags on the same line anywhere? Just kinda want to understand the implications... I guess this is ok though.